.github/workflows/ci.yml

```diff
@@ -21,22 +21,11 @@ jobs:
     - name: Run Infrastructure tests on BouncyCastle
       run: |
         dotnet test tests/DotNetLightning.Infrastructure.Tests -p:BouncyCastle=True
-    - name: Restore NSec and Secp256k1.Net
-      run: git checkout src/NSec src/Secp256k1.Net
-    - name: Clean to prepare for NSec build
-      run: |
-        dotnet clean
-    - name: Run tests
-      run: |
-        dotnet test
-    - name: Package NSec build
-      run: |
-        cd $GITHUB_WORKSPACE/src/DotNetLightning.Core
-        dotnet pack -p:Configuration=Release -p:Version=1.1.0-date`date +%Y%m%d-%H%M`.git-`echo $GITHUB_SHA | cut -c 1-7`
```

Comment:
is there a way to place a multi-line comment in YAML?

Reply:
No. Some googling suggests that only single-line comments are supported.

Reply:
alright :(